Source/monster.cpp

```diff
@@ -946,23 +946,23 @@ void PlaceQuestMonsters()
 
 		if (QuestStatus(Q_LTBANNER)) {
 			auto dunData = LoadFileInMem<uint16_t>("Levels\\L1Data\\Banner1.DUN");
-			SetMapMonsters(dunData.get(), 2 * setpc_x, 2 * setpc_y);
+			SetMapMonsters(dunData.get(), Point { setpc_x, setpc_y } * 2);

```

Comment:
This uses the Point operator*(float) or?
If that is the case this could result in different result (cause of rounding/truncation) or?
Same on other occurrences.

Reply:
Rounding/truncation with a flat 2? Would that really be possible?

Are you suggesting I should add an overload for the operator with ints?

Reply:
I would add a new overload.
There are also devices without FPU.